packages/flutter/lib/src/material/text_field.dart

```diff
@@ -604,6 +605,12 @@ class TextField extends StatefulWidget {
   /// [InputDecoration.enabled] property.
   final bool? enabled;
 
+  /// Determines whether this widget ignores pointer events.
+  ///
+  /// If true, this widget does not respond to pointer events,
+  /// If false this widget responds to all pointer events.
+  final bool? ignorePointers;
```

Comment:
nit: consider mentioning this is `null` by default.

Reply:
+1 to this suggestion.